client/components/Email/render.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import { ServerStyleSheet, StyleSheetManager } from 'styled-components';
+import juice from 'juice';
+import beautify from 'js-beautify';
+
+import { reset, globals } from './styles';
+
+const inlineStylesWithMarkup = (EmailMarkup: React.ReactNode, extraStyles: string) => {
```

Comment:
Any reason this is uppercase? In my opinion it should not be unless it is to indicate that the argument is, e.g. a JavaScript class, React component definition, Sequelize model, etc. (as opposed to an instance of these things)